.drone.star

```diff
@@ -106,6 +106,7 @@ def main(ctx):
         litmusOcisOldWebdav(),
         litmusOcisNewWebdav(),
         litmusOcisSpacesDav(),
+        virtualViews(),
```

Comment:
`virtualViews` doesn't work on edge atm. Just remove it.

Reply:
Ok. :+1: